packages/expo-file-system/src/FileSystem.ts

```diff
@@ -69,6 +69,15 @@ export async function readAsStringAsync(
   return await ExponentFileSystem.readAsStringAsync(fileUri, options || {});
 }
 
+export async function getContentUriAsync(
+  fileUri: string
+): Promise<string> {
+  if (!ExponentFileSystem.getContentUriAsync) {
```

Comment:
Should we return the same `fileUri` on iOS? Does iOS have similar mechanism?

Reply:
I think IOS could use expo-sharing to do the same work, but for android there's not such method to expose data to js layer. Maybe returning fileUri for IOS would be better.